src/components/EmailNotifications/EmailNotificationsForm.js

```diff
@@ -0,0 +1,295 @@
+            text-align: center;
+          `}
+        >
+          <div
+            css={`
+              align-items: center;
+            `}
+          >
+            <img
+              src={emailNotifcationIllustration}
+              width={141}
+              height={141}
+              alt=""
+            />
+          </div>
+          <span
+            css={`
+              ${textStyle('title2')};
+              margin-top: ${4 * GU}px;
+            `}
+          >
+            {existingEmail
+              ? `Update "${existingEmail}"`
+              : 'Stay up to date with email notifications'}
+          </span>
+
+          <TextContent update={existingEmail} account={account} theme={theme} />
+
+          <div
+            css={`
+              margin-top: ${5 * GU}px;
+            `}
+          >
+            <div
+              css={`
+                display: flex;
+                flex-direction: row;
+                justify-content: space-between;
+              `}
+            >
+              <Field
+                label={
+                  existingEmail
+                    ? 'Enter new email address'
+                    : 'Enter email address'
+                }
+                css={`
+                  width: 100%;
+                  margin-bottom: 0px;
+                `}
+              >
+                <TextInput
+                  value={emailAddress}
+                  adornment={
+                    emailInvalid ? (
+                      <IconCross
+                        css={`
+                          color: ${theme.negative};
+                        `}
+                      />
+                    ) : emailAddress.trim() ? (
+                      <IconCheck
+                        css={`
+                          color: ${theme.positive};
+                        `}
+                      />
+                    ) : (
+                      <IconCheck
+                        css={`
+                          opacity: 0;
+                        `}
+                      />
+                    )
+                  }
+                  adornmentPosition="end"
+                  type="email"
+                  wide
+                  onChange={handleEmailAddressChange}
+                  placeholder="email@address.com"
+                  onBlur={handleEmailAddressBlur}
+                />
+              </Field>
+
+              {!isModal && (
+                <div
+                  css={`
+                    display: flex;
+                    align-items: flex-end;
+                    margin-left: ${2 * GU}px;
+                  `}
+                >
+                  <Button
+                    mode="strong"
+                    disabled={emailInvalid || !termsAccepted}
+                    onClick={handleOnSubscribeToNotifications}
+                    size="medium"
+                  >
+                    {existingEmail ? 'Update' : 'Subscribe'}
+                  </Button>
```

Comment:
the form should wrap only the button? because i think that should go  wrap the email input, and when we are in modal mode we have two action buttons subscribe and opt out

Reply:
Yes I was thinking of wrapping the input, so that pressing the enter key would submit the form. For the modal mode, I think we could still have a form submitting the subscribe button since it has `mode="strong"`?

Reply:
ah yes sure, didn't know the guidelines regarding that